core/domain/exp_services.py

```diff
@@ -1396,7 +1849,16 @@ def reject_suggestion(editor_id, thread_id, exploration_id):
 
 def is_version_of_draft_valid(exp_id, version):
     """Checks if the draft version is the same as the latest version of the
-    exploration."""
+    exploration.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        version: str. The draft version.
+
+    Returns:
+        bool.True if the draft version is same as the the latest version of
```

Comment:
bool. Whether the given version number is the same as the current version number of the exploration in the datastore.